install/recorders.csv

```diff
@@ -422,6 +422,9 @@ CSI,CUSP3D,CUSP3D,42168,NSPS,21,251,0,0,2014-08-27T00:00:10Z,9999-01-01T00:00:00
 CSI,CUSP3D,CUSP3D,42169,IFPS,20,158,0,0,2014-08-12T00:05:00Z,9999-01-01T00:00:00Z
 CSI,CUSP3D,CUSP3D,42170,KOKS,20,118,0,0,2014-08-12T05:05:00Z,9999-01-01T00:00:00Z
 CSI,CUSP3D,CUSP3D,42172,NCHS,20,138,0,0,2014-08-27T00:00:10Z,9999-01-01T00:00:00Z
+CSI,CUSP3D,CUSP3D,42200,JSCB,21,252,0,-0.3,2014-06-15T00:00:00Z,9999-01-01T00:00:00Z
+CSI,CUSP3D,CUSP3D,42204,JSCB,22,252,0,-2.5,2014-06-15T00:00:00Z,9999-01-01T00:00:00Z
+CSI,CUSP3D,CUSP3D,42207,JSCB,23,252,0,-2.5,2014-06-15T00:00:00Z,9999-01-01T00:00:00Z
```

Comment:
From the docs, it looks like a CUSP-M4 is the model that is deployed.

I suggest calling it:

`CSI,CUSP-M4,3D-42200,JSCB,21,....`